src/demo-app/button/button-demo.html

```diff
@@ -1,92 +1,92 @@
 <div class="demo-button">
   <section>
-    <button md-button>flat</button>
-    <button md-raised-button>raised</button>
-    <button md-fab>
-      <md-icon class="md-24">check</md-icon>
+    <button mat-button>flat</button>
+    <button mat-raised-button>raised</button>
+    <button mat-fab>
+      <mat-icon class="mat-24">check</mat-icon>
```

Comment:
I think this is actually part of the Material icon css file, so we shouldn't change it.